test/negativebinomial.jl

```diff
@@ -17,4 +17,5 @@ end
 @testset "Check the corner case p==1" begin
     @test logpdf(NegativeBinomial(0.5, 1.0), 0) === 0.0
     @test logpdf(NegativeBinomial(0.5, 1.0), 1) === -Inf
+    @test iszero(rand(NegativeBinomial(0.5, 1.0)))
```

Comment:
Maybe create multiple samples just to be a bit more certain that it works as expected?
```suggestion
    @test all(iszero, rand(NegativeBinomial(rand(), 1.0), 10))
```